src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const listOfEmployees = document.querySelector('ul');
+
+function getEmployees(list) {
+  const employees = [...list.children];
+
+  return employees;
+}
```

Comment:
This function returns array of li element. It should return array of objects that looks like this instead:

> second, which returns an array of objects, where objects are employees.
> The schema for the employee:
> 
> ```
> {
>   name, 
>   position,
>   salary,
>   age
> }
> ```

Reply:
Mb you right, but i think, that immediately add the finished result is not a problem)

Reply:
`3. Create function getEmployees, which takes a list property, and returns an array of right objects.`
So yeah, let's do it that way